doc/articles/uno-development/ways-to-contribute.md

```diff
@@ -97,3 +97,12 @@ Once you're ready to create a PR, check out the [Guidelines for pull requests](.
 ### Need help?
 
 If there's anything you need, [ping us on our Discord Server](https://platform.uno/discord).
+
```

Comment:
Also @sasakrsmanovic I think we would want to fix this one also at the same time https://github.com/unoplatform/private/issues/354 with details regarding https://platform.uno/blog/github-sponsorship-matching-for-uno-platform-contributors-and-employees/

Reply:
We will do it in another PR as per feedback from @sasakrsmanovic